faithcomesbyhearing/dbp
# FCBH-1672 Arabic font for notes from old to new app flawed

# Description
- Fix wrong `charset`, `collation` values on `dbp_users_v2` database connection
- This fixes new v2->v4 entries

## Issue Link
Story: [FCBH-1672](https://fullstacklabs.atlassian.net/browse/FCBH-1672)  

## How Do I QA This
- Run `php artisan syncV2:notes` and verify that the notes are now synced correctly

## Checklist
